src/logger/POSTRequestLogger.ts

```diff
@@ -0,0 +1,143 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Logger, LogLevel } from '../index';
+import IntervalScheduler from '../scheduler/IntervalScheduler';
+
+export default class POSTRequestLogger implements Logger {
+  private logCapture: string[] = [];
+  private logSequenceNumber: number = 0;
+  private lock = false;
+  private static BATCH_SIZES = 75;
+  private intervalScheduler: IntervalScheduler;
+
+  constructor(
+    private name: string,
+    private meetingId: string,
+    private attendeeId: string,
+    private intervalMs: number,
+    private baseURL: string,
+    private level = LogLevel.WARN,
+  ) {
+    this.intervalScheduler = new IntervalScheduler(this.intervalMs);
+    var _this = this;
+    window.addEventListener('unload', function(event) {
+      _this.intervalScheduler.stop();
+      if(!navigator.sendBeacon){
+        return
+      }
+      let body = JSON.stringify({
```

Comment:
Please go over all ```let``` in this file and replace them with ```const``` if you do not assign a new value to it.